src/org/labkey/test/tests/core/admin/ShortUrlTest.java

```diff
@@ -0,0 +1,156 @@
+package org.labkey.test.tests.core.admin;
+
+import org.apache.commons.lang3.SystemUtils;
+import org.jetbrains.annotations.NotNull;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.Connection;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.TestProperties;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.BVT;
+import org.labkey.test.pages.admin.ShortUrlAdminPage;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.params.list.IntListDefinition;
+import org.labkey.test.util.DataRegionTable;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.labkey.test.WebTestHelper.buildRelativeUrl;
+
+@Category({BVT.class}) // TODO: switch to Daily
+public class ShortUrlTest extends BaseWebDriverTest
+{
+    private static final String PROJECT_NAME = "ShortUrlTest Project";
+    private static final String URL_PREFIX = "surl_test_";
+    private static final String URL_SUFFIX = ".url";
+    private static final String USER = "shorturl_user@shorturltest.test";
+    private static final AtomicInteger count = new AtomicInteger();
```

Comment:
AtomicInteger seems overkill here though I don't think it's doing any real harm. Why not just a simple `int`?